src/main/java/ru/svgogin/service/spark/handler/RestResponseEntityExceptionHandler.java

```diff
@@ -35,4 +41,37 @@
     return handleExceptionInternal(ex, bodyOfResponse,
         new HttpHeaders(), HttpStatus.NOT_FOUND, request);
   }
+
+  @ExceptionHandler(value = {ConstraintViolationException.class})
+  protected ResponseEntity<Object> handleNotValid(RuntimeException ex, WebRequest request) {
+    String message = ex.getMessage();
+    var bodyOfResponse = new ErrorDto(ErrorDto.ErrorCode.ERROR003, message);
+    log.warn(message);
+    return handleExceptionInternal(ex, bodyOfResponse,
+        new HttpHeaders(), HttpStatus.BAD_REQUEST, request);
+  }
+
+  @NonNull
+  @Override
+  protected ResponseEntity<Object> handleMethodArgumentNotValid(MethodArgumentNotValidException ex,
+                                                                @NonNull HttpHeaders headers,
+                                                                @NonNull HttpStatus status,
+                                                                @NonNull WebRequest request) {
+    var error = ex.getBindingResult().getFieldErrors().iterator().next();
+    var errorField = error.getField();
+    //This is a bullshit implementation, but I have no idea, how to improve it
```

Comment:
it is better to do it this way:
```java
  protected ResponseEntity<Object> handleMethodArgumentNotValid(MethodArgumentNotValidException ex,
                                                                @NonNull HttpHeaders headers,
                                                                @NonNull HttpStatus status,
                                                                @NonNull WebRequest request) {
    var error = ex.getBindingResult().getFieldErrors().iterator().next();
    var errorField = error.getField();
    var errorCode = errorCodeBy(error);
    var message = String.format("%s (%s)", errorCode.message, errorField);
    log.warn(message);
    var bodyOfResponse = new ErrorDto(errorCode, message);
    return handleExceptionInternal(
        ex, bodyOfResponse, headers, status, request
    );
  }

  private ErrorCode errorCodeBy(FieldError fieldError) {
    var notBlankExists = Optional.ofNullable(fieldError.getCodes())
        .map(codes -> Set.of(codes).contains("NotBlank"))
        .orElse(false);
    if (notBlankExists) {
      return ErrorCode.ERROR004;
    } else {
      return ErrorCode.ERROR003;
    }
  }
```
But also you shouldn't return result only for first error

Reply:
Done